wire-runtime-swift/src/main/swift/Codable/ProtoEnum.swift

```diff
@@ -18,7 +18,7 @@ import Foundation
 /// Common protocol that all Wire generated enums conform to
 /// - Note: All ProtoEnums will convert to/from their field and string equivalent when serializing via Codable.
 /// This matches the Proto3 JSON spec: https://developers.google.com/protocol-buffers/docs/proto3#json
-public protocol ProtoEnum : LosslessStringConvertible, Codable {
+public protocol ProtoEnum : LosslessStringConvertible, Codable, ProtoCodable {
```

Comment:
This is _mildly_ breaking in that you're forced to recompile, but that seems fine / is desirable